app/views/layouts/listnav/_container_image.html.haml

```diff
@@ -44,6 +44,16 @@
                 = link_to("#{ui_lookup(:table => "container_groups")} (#{num_groups})",
                     {:action => 'show', :id => @record, :display => 'container_groups'},
                      :title  => _("View %s") % ui_lookup(:table => "containers"))
+          - if role_allows(:feature => "container_node_show")
+            - num_nodes = @record.number_of(:container_nodes)
+            - if num_nodes == 0
+              %li.disabled
+                = link_to("#{ui_lookup(:table => "container_nodes")} (0)", "#")
+            - else
+              %li
+                = link_to("#{ui_lookup(:table => "container_nodes")} (#{num_nodes})",
+                    {:action => 'show', :id => @record, :display => 'container_nodes'},
+                     :title  => _("View %s") % ui_lookup(:table => "container_nodes"))
```

Comment:
titles on the left should be plural, use tables in ui_lookup call, like this: ui_lookup(:tables => "container_nodes")